requirements.txt

```diff
@@ -1,17 +1,6 @@
-Pillow==10.3.0
 PySMT==0.9.0
 asdl-adt==0.1.0
 asdl==0.1.5
-astor==0.8.1
 build==0.7.0
-black==24.3.0
-coverage==6.3.2
-jupyter-contrib-nbextensions==0.5.1
-numpy==1.23.4
-pre-commit==3.6.0
-pytest-cov==3.0.0
-pytest-xdist==3.0.2
-pytest==7.1.1
-tox==3.24.5
-yapf==0.32.0
 z3-solver==4.12.6.0
```

Comment:
Can we bump z3 to 4.13.0.0? I am actually not sure why dependabot didn't do that...

Reply:
Done!